src/Domain/LeanCode.DomainModels.EF/TypedIdConverter.cs

```diff
@@ -0,0 +1,25 @@
+using LeanCode.DomainModels.Ids;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+namespace LeanCode.DomainModels.EF;
+
+[System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
+public class PrefixedTypedIdConverter<TId> : ValueConverter<TId, string>
+    where TId : struct, IPrefixedTypedId<TId>
+{
+    public static readonly PrefixedTypedIdConverter<TId> Instance = new();
+
+    private PrefixedTypedIdConverter()
```

Comment:
Making the ctors `public` would allow the converters to be used when configuring DB conventions.

Reply:
I fear it won't allow that - there is too much magic going on. If conventions _will_ be available, they should be supported by the CoreLib.

Reply:
W8, does conversions are configured _per type_ either way?

Reply:
> If conventions will be available, they should be supported by the CoreLib.

I don't get what you mean here. In order to use `builder.Properties<Type>().HaveConversion<TypeConverter>();` the `TypeConverter` must expose public argless ctor.

> W8, does conversions are configured per type either way?

Yes, `DbContext` conventions in `ConfigureConventions` work per type.